pyface/ui/wx/fields/field.py

```diff
@@ -35,28 +35,6 @@ class Field(MField, Widget):
     #: An optional context menu for the field.
     context_menu = Instance("pyface.action.menu_manager.MenuManager")
 
-    # ------------------------------------------------------------------------
-    # IField interface
-    # ------------------------------------------------------------------------
-
-    def _initialize_control(self):
-        """ Perform any toolkit-specific initialization for the control. """
-        self.control.SetToolTip(self.tooltip)
-        self.control.Enable(self.enabled)
-        self.control.Show(self.visible)
```

Comment:
it's not clear to me why `Field.initialize_control` can be removed here nor do i understand why `Field._create` and `Field.destroy` can be removed either.

Reply:
These are now handled without any toolkit-specific code in the base `MField` class (via `_set_control_tooltip`, etc.)

The `_create` and `_destroy` aren't needed because their only purpose was to call `_add_event_listeners` and `_remove_event_listeners` which is now done in the `_create` and `_destroy` methods of `MLayoutWidget`.